donate/templates/fragments/campaign_page_fundraiseup_form.html

```diff
@@ -0,0 +1,12 @@
+{% load static i18n %}
+
+<div class="layout__half-col {{ order }}">
+    <div class="donate-form">
+        <h2 class="heading heading--primary donate-form__heading-mobile">
+            {% trans page.intro_header context "Header" %}
+        </h2>
+        <div class="donate-form__fundraiseup-form">
```

Comment:
Let's replace this with `{% include %}` to donate/templates/fragments/donate_form_fundraise_up.html

Reply:
done!